resources/keywords/provision_dbinstance.resource

```diff
@@ -36,10 +44,20 @@ Toggle Database Provider Drop Down ${isv_name}
     ...    Toggle RDS Database ${engine} Drop Down
 
 Toggle RDS Database ${engine} Drop Down
-    Select From List By Label    ${engineTypeDrpDwn}    Select one
-    Select From List By Label    ${engineTypeDrpDwn}    ${engine}
+    #Select From List By Label    ${engineTypeDrpDwn}    Select one
```

Comment:
@ikanias Please remove the commented keywords and keep the comments. Do we need 15s x 2s wait time for the value on list to be selected?

Reply:
I see what you are saying. I just wanted to leave the commented keywords so we would know how to change back when the bug will be fixed. This is temporary.

Reply:
@ikanias Can you please review as the bug marked as done and unresolved. And Please review the wait time.

Reply:
OK will check.